requirements.txt

```diff
@@ -1,5 +1,3 @@
-docker-compose==1.7.0
-docker-py==1.8.0
-Jinja2==2.8
-PyYAML==3.11
-requests==2.7.0
+docker-compose>=1.7.0
+Jinja2>=2.8
+PyYAML>=3.11
```

Comment:
Hard to argue what "most people" would do nowadays. :)

That said: we shouldn't preclude our friends in Fedora-land from carrying a patched RPM if they see fit -- but we should be *very* clear, as upstream, what versions of these things we support. And for now, we want to pin these versions.